java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -184,6 +185,7 @@ function findJDK(onChange: (path : string | null) => void): void {
     }
 
     let currentJdk = find();
+    validateJDKCompatibility(currentJdk);
```

Comment:
note -- there's a pending PR #7497 that introduces `projectJDK`. Depending on which one will merge first, the other PR should change to validate the `projectJDK`